metric/cumulative_test.go

```diff
@@ -83,6 +84,62 @@ func TestCumulative(t *testing.T) {
 	}
 }
 
+func TestCumulativeConstLabel(t *testing.T) {
+	r := NewRegistry()
+
+	f, _ := r.AddFloat64Cumulative("TestCumulativeWithConstLabel",
+		WithLabelKeys("k1"),
+		WithConstLabel(map[metricdata.LabelKey]metricdata.LabelValue{
+			{Key: "const"}:  metricdata.NewLabelValue("same"),
+			{Key: "const2"}: metricdata.NewLabelValue("same2"),
+		}))
+
+	e, _ := f.GetEntry(metricdata.LabelValue{})
+	e.Inc(5)
+	e, _ = f.GetEntry(metricdata.NewLabelValue("k1v1"))
+	e.Inc(1)
+	e, _ = f.GetEntry(metricdata.NewLabelValue("k1v1"))
```

Comment:
nit: e is not used again. Please remove it.

Reply:
remove :)